packages/react-components/react-positioning/src/utils/parsePopperPlacement.ts

```diff
@@ -0,0 +1,15 @@
+import type { BasePlacement, Placement, Variation } from '@popperjs/core';
+
+/**
+ * Parses Popper placement and returns the different components
+ * @param placement - the Popper.js placement (i.e. bottom-start)
+ *
+ * @returns side and alignment components of the placement
+ */
+export function parsePopperPlacement(placement: Placement): { basePlacement: BasePlacement; alignment: Variation } {
+  const tokens = placement.split('-');
+  return {
```

Comment:
Is there any way that the placement could be malformed and not have 2 tokens?  If so, consider a check here to either throw a specific error, provide a default, or return null.